.eslintrc.js

```diff
@@ -17,6 +17,7 @@ module.exports = {
   },
   rules: {
     '@typescript-eslint/no-unsafe-assignment': 'off',
+    '@typescript-eslint/no-unsafe-argument': 'off',
```

Comment:
To handle the pervasive use of implicit any everywhere.  We should follow-up with proper typing but out-of-scope of this PR as it is an existing issue and scope is narrowed down to dependency management at this time.